palindrom

```diff
@@ -0,0 +1,34 @@
+function pal(start , end , thisword){
+
+   
+    if( thisword[start] != thisword[end] ){
+        console.log(thisword[start]);
+        console.log(thisword[end]);
+        return false;
+    }
+   
+    if(start != end){
+        pal(start+1 , end-1 , thisword);
+        console.log("plaindrom");
+    }
+    
+    return true;
+}
+
+function isPalindromeFunction(str)
+{
```

Comment:
By convention, blocks in JavaScript have their opening curly braces on the same line.

```js
function isPalindrome(str) {
  // your implementation
}
```